disconnected/updating/index.adoc

```diff
@@ -1,42 +1,36 @@
 
 toc::[]
 
-////
-WARNING: This assembly has been moved into a subdirectory for 4.14+. Changes to this assembly for earlier versions should be done in separate PRs based off of their respective version branches. Otherwise, your cherry picks may fail.
-
-To do: Remove this comment once 4.13 docs are EOL.
-////
-
 A disconnected environment is one in which your cluster nodes cannot access the internet.
 For this reason, you must populate a registry with the installation images.
 If your registry host cannot access both the internet and the cluster, you can mirror the images to a file system that is disconnected from that environment and then bring that host or removable media across that gap.
 If the local container registry and the cluster are connected to the mirror registry's host, you can directly push the release images to the local registry.
 
 A single container image registry is sufficient to host mirrored images for several clusters in the disconnected network.
 
-[id="about-disconnected-updates-mirroring"]
+[id="about-disconnected-updates-mirroring_{context}"]
 == Mirroring {product-title} images
 To update your cluster in a disconnected environment, your cluster environment must have access to a mirror registry that has the necessary images and resources for your targeted update. The following page has instructions for mirroring images onto a repository in your disconnected cluster:
 
-* xref:../../../updating/updating_a_cluster/updating_disconnected_cluster/mirroring-image-repository.adoc#mirroring-ocp-image-repository[Mirroring {product-title} images]
+* xref:../../disconnected/updating/mirroring-image-repository.adoc#mirroring-ocp-image-repository[Mirroring {product-title} images]
 
-[id="about-disconnected-updates-update"]
+[id="about-disconnected-updates-update_{context}"]
 == Performing a cluster update in a disconnected environment
 
 You can use one of the following procedures to update a disconnected {product-title} cluster:
 
-* xref:../../../updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc#updating-restricted-network-cluster-OSUS[Updating a cluster in a disconnected environment using the OpenShift Update Service]
+* xref:../../disconnected/updating/disconnected-update-osus.adoc#updating-disconnected-cluster-OSUS[Updating a cluster in a disconnected environment using the OpenShift Update Service]
 
-* xref:../../../updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update.adoc#updating-restricted-network-cluster[Updating a cluster in a disconnected environment without the OpenShift Update Service]
+* xref:../../disconnected/updating/disconnected-update.adoc#updating-disconnected-cluster[Updating a cluster in a disconnected environment without the OpenShift Update Service]
 
-[id="about-disconnected-updates-uninstalling-osus"]
+[id="about-disconnected-updates-uninstalling-osus_{context}"]
 == Uninstalling the OpenShift Update Service from a cluster
```

Comment:
Pre-existing nit: it seems unbalanced to talk about OSUS uninstall here as a top-level section, while OSUS install is tucked down under *Performing a cluster update in a disconnected environment*.  You can punt as orthogonal to your change, but maybe this restructure gives us a chance to rebrand the current *Updating a cluster in a disconnected environment using the OpenShift Update Service* section as a *Managing an OpenShift Update Service* section or something?  Because it's really aimed at the OSUS admins, with a small bit of once-per-cluster configuration, and then a *Next steps* section that basically says "now you can do all the usual connected update stuff" for the for-each-cluster-update part.

Reply:
This actually sounds like an interesting idea and one that I'd personally be willing to implement. I agree that calling the OSUS page "managing OSUS" would make more sense given that the procedures there have nothing to do with actually performing the updates themselves.

For now, I think I'll focus on migrating the content over, and later I'll circle back to do the reframing